src/components/ItemTable/ItemTable.tsx

```diff
@@ -17,21 +17,25 @@ const ItemTable = ({ itemTableData }: ItemTableProps) => {
       <Table
         columnHeaders={itemTableData.tableHeadings}
         tableData={itemTableData.tableData}
+        mt="s"
         mb="s"
-        // TODO: These styles approximate those of the old app.
-        // Design will VQA the component with the DS defaults, but leaving this for reference.
-        // sx={{
-        //   tableLayout: "fixed",
-        //   width: "full",
-        //   tr: { border: "0 !important", padding: 0 },
-        //   "th, td, th > span, td > span": {
-        //     paddingTop: "xs",
-        //     paddingBottom: "xs",
-        //   },
-        // }}
+        sx={{
```

Comment:
We decided a while back if there's more than one line of inline styles to move it to a CSS module. Can you put this in another file? Also, can you replace 32px with nypl-space-l token?

Reply:
Updated.